traffic_ops/traffic_ops_golang/topology/validation.go

```diff
@@ -54,6 +56,45 @@ func checkForSelfParents(nodes []tc.TopologyNode, index int) error {
 	return nil
 }
 
+// checkForEdgeParents returns an error if an index given in the parents array, adds a warning + returns a nil error if
```

Comment:
The first part of this comment `returns an error if an index given in the parents array, ...` doesn't make sense.